ui/raidboss/data/07-dt/raid/r3n.ts

```diff
@@ -1,14 +1,175 @@
+    },
+    {
+      id: 'R3N Knuckle Sandwich',
+      type: 'StartsUsing',
+      netRegex: { id: '93D5', source: 'Brute Bomber', capture: true },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'R3N Brutal Lariat 9AD4',
+      type: 'StartsUsing',
+      netRegex: { id: '9AD4', source: 'Brute Bomber', capture: false },
+      response: Responses.goEast(),
+    },
+    {
+      id: 'R3N Brutal Lariat 9AD5',
+      type: 'StartsUsing',
+      netRegex: { id: '9AD5', source: 'Brute Bomber', capture: false },
+      response: Responses.goWest(),
+    },
+    {
+      id: 'R3N Murderous Mist',
+      type: 'StartsUsing',
+      netRegex: { id: '93B5', source: 'Brute Bomber', capture: false },
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'R3N Barbarous Barrage',
+      type: 'StartsUsing',
+      netRegex: { id: '93B2', source: 'Brute Bomber', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Knockback Towers',
+        },
+      },
+    },
+    {
+      id: 'R3N Fire Spin Clockwise',
+      type: 'StartsUsing',
+      netRegex: { id: '93D0', source: 'Brute Bomber', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: Outputs.clockwise,
+      },
+    },
+    {
+      id: 'R3N Fire Spin Counterclockwise',
+      type: 'StartsUsing',
+      netRegex: { id: '93D1', source: 'Brute Bomber', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: Outputs.counterclockwise,
+      },
+    },
+    {
+      id: 'R3N Fuses of Fury',
+      type: 'StartsUsing',
+      netRegex: { id: '93B6', source: 'Brute Bomber', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Long => Short',
+        },
+      },
+    },
+    {
+      id: 'R3N Lariat Combo East to West',
+      type: 'StartsUsing',
+      netRegex: { id: '9ADC', source: 'Brute Bomber', capture: false },
+      durationSeconds: 15,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'East, then West',
+        },
+      },
+    },
+    {
+      id: 'R3N Lariat Combo East to East',
+      type: 'StartsUsing',
+      netRegex: { id: '9ADD', source: 'Brute Bomber', capture: false },
+      durationSeconds: 15,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'East, then East',
+        },
+      },
+    },
+    {
+      id: 'R3N Lariat Combo West to East',
+      type: 'StartsUsing',
+      netRegex: { id: '9ADE', source: 'Brute Bomber', capture: false },
+      durationSeconds: 15,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'West to East',
```

Comment:
```suggestion
          en: 'West, then East',
```
Probably best to ensure the style is consistent with the East -> West trigger.